### PR TITLE
Teamu 1099 Deploy CCCV demo version

### PR DESCRIPTION
**Pull request by wtcarter**
Deploys first demo version to dev, testing pipeline

### CODE REVIEW

File: packages/cccv/src/components/MapStyleSelector/MapStyleSelector.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+.map-style-selector {}
```
Comment (smozely) [inline packages/cccv/src/components/MapStyleSelector/MapStyleSelector.scss:1]: No op?

File: packages/cccv/src/global.d.ts [hunk 2]
```diff
@@ -0,0 +1,221 @@
+import _ from 'lodash';
```
Comment (smozely) [inline packages/cccv/src/global.d.ts:1]: Has this got a lot that was just copy/paste from plan limits?

File: packages/cccv/src/values/councils.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import gwrcLogo from '@images/gwrc-logo-header.svg';
```
Comment (smozely) [inline packages/cccv/src/values/councils.tsx:1]: Is this needed, seems like copy / paste from plan limits?

**Review by smozely — COMMENTED** [review]
He @wtcarter overall looks pretty good, but just lots of small comments mainly around unused code, or code thats been copy / pasted from the plan limits app

**Comment by wtcarter** [comment]
Ah, okay.  Thanks for that.  I think when I started, I wasn’t sure what I was going to need :)

Wayne
________________________________

File: packages/cccv/README.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```
[the remark this answers is not in the document]
Comment (wtcarter) [reply packages/cccv/README.md:1]: Yes, for sure

File: packages/cccv/src/components/FarmManagementUnit/FarmManagementUnit.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import FarmManagementUnitProps from "@src/models/FarmManagementUnit.ts";
+import "./FarmManagementUnit.scss";
+
+const FarmManagementUnit = ( { id, fmuName1, ecoliBase, ecoliObj, byWhen }: FarmManagementUnitProps ) => {
+  return <div className="farmManagementUnit space-y-0 p-6 h-full bg-gray-700" id={`fmu_${id}`}>
+    <h2 className={"space-x-0 space-y-0"}>{fmuName1}</h2>
```
[the remark this answers is not in the document]
Comment (wtcarter) [reply packages/cccv/src/components/FarmManagementUnit/FarmManagementUnit.tsx:6]: An artifact from previous versions tbh.  Will prob disappear

File: packages/cccv/src/components/FarmManagementUnit/FarmManagementUnit.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    <h4>Overview</h4>
+    <h4>Outcomes</h4>
+    <h3>E. coli</h3>
+    <ol className={'space-y-0'}>
```
[the remark this answers is not in the document]
Comment (wtcarter) [reply packages/cccv/src/components/FarmManagementUnit/FarmManagementUnit.tsx:10]: As above

**Pull request closed without merging**